dist/main.js

```diff
@@ -0,0 +1,218 @@
+/*
```

Comment:
Please add a professional README as it's the first point of interaction to the project user.